src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -643,28 +643,23 @@ impl<'a, 'tcx> LinkCollector<'a, 'tcx> {
                 if ns != Namespace::ValueNS {
```

Comment:
FYI the side channel comment just above this line seems to be out of date.

Reply:
Hmm? I don't think so, the `(DefKind, DefId)` pair it returns is for storing in a side channel.

Reply:
Ah, okay.